include/deal.II/distributed/fully_distributed_tria.h

```diff
@@ -0,0 +1,75 @@
+#ifndef dealii_fully_distributed_tria_h
+#define dealii_fully_distributed_tria_h
+
+namespace parallel
+{
+  namespace fullydistributed
+  {
+
+    template <int dim>
+    struct CellData
```

Comment:
I think this all needs to be rebased :-)

Reply:
I have rebased. Please not the workaround in `CellData<dim>::serialize`!